pypesto/objective/aggregated.py

```diff
@@ -206,6 +161,48 @@ def reset_steadystate_guesses(self):
                 objective.reset_steadystate_guesses()
 
 
+def aggregate_results(rvals):
+
+    # sum over fval/grad/hess
+    result = {
+        key: sum(rval[key] for rval in rvals)
+        for key in [FVAL, CHI2, SCHI2, GRAD, HESS, HESSP]
+        if rvals[0].get(key, None) is not None
+    }
+
+    # extract rdatas and flatten
+    result[RDATAS] = []
+    for rval in rvals:
+        if RDATAS in rval:
+            result[RDATAS].extend(rval[RDATAS])
```

Comment:
should it be extend or append?

Reply:
extend. `AggregateObjective` is still an `Objective`, so it should follow the same output API and result[RDATAS] should be a list of ReturnData and not a list of lists.